src/eloqua.js

```diff
@@ -1,3 +1,7 @@
+/*
+* @version X.X.X
+*/
```

Comment:
We actually do have a LICENSE file committed, it's in the develop branch. But yeah I think you're right, we should have that extra stuff here and in the readme.

Also quick sanity check, @brianpabent, is MIT license okay for this?